vignettes/epidist.Rmd

```diff
@@ -297,38 +299,57 @@ data.frame(
   tab_caption("(ref:pars)")
 ```
 
+For example, we can use the built in `summary()` function
+
+```{r}
+summary(fit)
+```
+
+to see that the model has converged and recovered the delay distribution distribution paramters reasonably well.
+
 The `epidist` package also provides functions to make common post-processing tasks easy.
 For example, individual predictions of the lognormal delay parameters can be extracted using:
 
 ```{r}
 pred <- predict_delay_parameters(fit)
 ```
 
-Figure \@ref(fig:fitted-lognormal) shows the lognormal delay distribution obtained using the average of the `mu` and `sigma` draws.
+Figure \@ref(fig:fitted-lognormal) shows the lognormal delay distribution obtained for 100 draws from the posterior distribution of the `mu` and `sigma` parameters.
 Whereas in Figure \@ref(fig:obs-est) the histogram of censored, truncated, sampled data was substantially different to the underlying delay distribution, using `epidist()` we have obtained a much closer match to the truth.
 
-(ref:fitted-lognormal) A fitted delay distribution (in pink) as compared with the true delay distribution (in black).
+(ref:fitted-lognormal) The fitted delay distribution (pink lines, 100 draws from the posterior) compared to the true underlying delay distribution (black line). Each pink line represents one possible delay distribution based on the fitted model parameters.
 
 ```{r fitted-lognormal, fig.cap="(ref:fitted-lognormal)"}
+# Sample 100 random draws from the posterior
+set.seed(123)
+sample_draws <- sample(nrow(pred), 100)
+
 ggplot() +
+  # Plot the true distribution
   geom_function(
     data = data.frame(x = c(0, 30)),
     aes(x = x),
     fun = dlnorm,
+    linewidth = 2,
     args = list(
       meanlog = secondary_dist[["mu"]],
       sdlog = secondary_dist[["sigma"]]
     )
   ) +
-  geom_function(
-    data = data.frame(x = c(0, 30)),
-    aes(x = x), fun = dlnorm,
-    args = list(
-      meanlog = mean(pred$mu),
-      sdlog = mean(pred$sigma)
-    ),
-    col = "#CC79A7"
-  ) +
+  # Plot 100 sampled posterior distributions
```

Comment:
This was also a sense check but I've left it in as I think it might be useful

Reply:
Yeah I think it's better, thanks. IMO the black line is too thick -- maybe 1.5 / I would try to distinguish in some other way but it's minor